### DISCUSSION TITLE
Could not connect to the remote site.

### DISCUSSION THREAD

**Discussion by beemerrider** [akeeba/panopticon#832] [Q&A (Community Support)]
I have 24 sites successfully added to Panopticon, mainly Joomla and a few WP sites all working great. I have two Joomla sites I cannot connect to though. All sites are on the same dedicated server. No MOD security rules are being tripped. Setup on Joomla sites with firewall, SEO, etc, are all similar. I have rebuilt the htaccess file with Admin tools. There is no record of the firewall blocking the request. Below is the debug info:

Could not connect to the remote site.

Instead, we got an error which we should be able to provide troubleshooting information for (Akeeba\Panopticon\Exception\SiteConnection\APIApplicationIsBroken), but we don't have the code for it. This should have not happened.

Please delete all files and folders inside the tmp folder of Akeeba Panopticon. This addresses the problem of this page's cached copy being severely out of date. If this doesn't help, please let us know.

Troubleshooting information

Process step: Unauthenticated access (can I even access the API at all?)

HTTP Status	500
HTTP Body	
{"errors":[{"code":500,"title":"Internal server error"}]}
HTTP HEADERS	
Date
Wed, 23 Oct 2024 23:47:32 GMT
Server
Apache
x-frame-options
SAMEORIGIN
referrer-policy
strict-origin-when-cross-origin
cross-origin-opener-policy
same-origin
strict-transport-security
max-age=63072000
X-Powered-By
JoomlaAPI/1.0
Expires
Wed, 17 Aug 2005 00:00:00 GMT
Cache-Control
no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Pragma
no-cache
Last-Modified
Wed, 23 Oct 2024 23:47:32 GMT
Connection
close
Transfer-Encoding
chunked
Content-Type
application/vnd.api+json; charset=utf-8

**Comment by jzappe** [comment]
Have you activated the error display? I get the same error.

On my website the plugin “ChronoG3_Plg” as part of the component “Chronoforms 7” seems to cause the HTTP 500 error:
Error: Class \"G3\\L\\AppLoader\" not found in \/var\/www\/vhosts\/www\/plugins\/system\/chronog3_plg\/chronog3_plg.php:111

Deleting the files in the tmp folder did not help.
